### DISCUSSION TITLE
Remove NKeys Implementation?

### DISCUSSION THREAD

**Discussion by rickdotnet** [nats-io/nats.net#555] [General]
I was doing some research on NKeys and originally started looking at the code in this repo instead of [that repo](https://github.com/nats-io/nkeys.net/), which looks to be the future home for the NKey bits.

Are there current plans to move [this code](https://github.com/nats-io/nats.net.v2/blob/main/src/NATS.Client.Core/NKeyPair.cs) (and more) out and replacing it with a reference to the new library?

The last traction I see on it is the [discussion here](https://github.com/nats-io/nats.net.v2/issues/309)

**Comment by mtmk** [comment]
yes that makes sense 💯 it's part of NATS Auth* stuff in general. I'm still working on the https://github.com/nats-io/jwt.net repo on and off. Once that's ready (should be soon) then we can release a stable NATS.NKeys version (I think @mkobaly has suggestion for the API) we can replace the nkeys and crypto bits and pieces from this repo.

**Comment by rickdotnet** [comment]
Looks like we're all charting into the same territory. I appreciate the effort y'all are putting in to get these updates in quickly.
